src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -163,8 +163,11 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             src="{{workItem.relationalData.creator.attributes.imageURL + '&s=20'}}"
             id="WI_details_reporter_img" />          
           <div class="pull-left marginR10">                          
-              <span id="WI_details_reporter_user"> 
-                {{workItem.relationalData?.creator?.attributes.fullName || 'No creator found'}} 
```

Comment:
Checked on demo.almighty.io, the UI behaves correctly. As full name is null it displays "No Creator". Core needs to send the Github name if the user's full name is not available.